validation/chOpenApiValidator.go

```diff
@@ -246,3 +233,37 @@ func handleParseError(pe *openapi3filter.ParseError) models.CHError {
 func findRoute(r routers.Router, req *http.Request) (route *routers.Route, pathParams map[string]string, err error) {
 	return r.FindRoute(req)
 }
+
+func loadSchemaFromFile(ctx context.Context, openApiSpec, contextId string) (*openapi3.T, error){
```

Comment:
Just from an ordering, maybe move this to after the getSchema function

Reply:
Reordered as Matt suggested